src/test/java/test/listeners/github2385/TestMultiInherit.java

```diff
@@ -0,0 +1,14 @@
+package test.listeners.github2385;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertTrue;
+
+public class TestMultiInherit implements ITestClassInterface, ITestInterface{
```

Comment:
By undocumented convention, we use the term "Sample" when it is possible (in order to make a distinction between the testng tests and the test case tests)

Reply:
Sample added